jobs/kpi-forecasting/kpi_forecasting/configs/search_forecasting_daily_active_users.yaml

```diff
@@ -0,0 +1,51 @@
+---
+metric_hub:
+  app_name: "multi_product"
+  slug: "search_forecasting_daily_active_users"
+  alias: "search_forecasting_daily_active_users"
+  start_date: "2018-01-01"
+  end_date: NULL
+
+forecast_model:
+  model_type: "prophet"
+  segment_settings:
+    desktop:
+      start_date: "2018-01-01"
+      end_date: NULL
+      holidays: ["easter", "covid_sip11"]
+      regressors: ["post_esr_migration", "in_covid"]
+      parameters:
+        changepoint_prior_scale: [0.001, 0.01, 0.1, 0.2, 0.5]
+        changepoint_range: [0.8, 0.9]
+        weekly_seasonality: True
+        yearly_seasonality: True
+        cv_settings:
```

Comment:
I think the value of cv_settings might still be a dotmap when it gets unpacked in kpi_forecasting

Reply:
Ah, good catch. I'll change that here for consistency.